develop/devguide/ClassLibrary/ClassLibraryInterAppClasses.md

```diff
@@ -95,9 +95,9 @@ Begin by checking and applying everything to meet the DataMiner, DIS, protocol a
 Now there are 4 steps for development:
 
 1. Creating an API with messages that define the data you want to share.
-1. Creating executors that define how to process a message.
-1. Parsing a received call.
-1. Sending a call.
+2. Creating executors that define how to process a message.
```

Comment:
Ideally, each entry of a numbered list should start with "1. ". This way, you avoid extra editing when something is added or deleted in a list. See https://docs.dataminer.services/CONTRIBUTING.html#lists